pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native platform command line client.
+> More information: <https://native.electrode.io/reference/index-6>.
+
+- Create a new `ern` application (`MiniApp`):
+
+`ern create-miniapp {{appName}}`
+
+- Run one or more `MiniApps` in the iOS Runner application:
+
+`ern run-ios`
+
+- Run one or more `MiniApps` in the Android Runner application:
+
+`ern run-android`
```

Comment:
```suggestion
- Run one or more `MiniApps` in the iOS/Android Runner application:

`ern run-{{ios|android}}`
```

Two examples could be joined.

Reply:
We might use `iOS / Android` (with spaces) or `iOS or Android` though.

Reply:
Great point, consolidated the two examples into one.